### PR TITLE
changes after a cluster installation with a customer

### PR DESCRIPTION
**Pull request by ecuriotto**
After using this document to install a cluster, I'm proposing these updates.

### CODE REVIEW

File: md/install-a-bonita-bpm-cluster.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Install a Bonita BPM cluster
+# Install a Bonita cluster
```
Comment (educhastenier) [inline md/install-a-bonita-bpm-cluster.md]: no, the product has changed name in 7.6+

File: md/install-a-bonita-bpm-cluster.md [hunk 2]
```diff
@@ -38,16 +38,21 @@ user at the least: `"ec2:DescribeInstances"`.
 
 In this part we will create a cluster from scratch. We will initialize the database on which the cluster will run, then we will configure nodes to run on this cluster.
 
-### <a id="create_init_bonita_db" /> Create and initialize the database for Bonita BPM Platform
+## Allow communication between servers
+Bonita cluster uses Hazelcast as the distributed cluster dispatcher layer.
+Hazelcast needs the port 5701 to be opened on all the servers that compose the cluster.
```
Comment (educhastenier) [inline md/install-a-bonita-bpm-cluster.md]: Isn't this port configurable? If so, do not hard-code it.

File: md/install-a-bonita-bpm-cluster.md [hunk 3]
```diff
@@ -38,16 +38,21 @@ user at the least: `"ec2:DescribeInstances"`.
...
+
+In this step you will create and initialize the database for the Bonita Platform cluster using the [platform setup tool](BonitaBPM_platform_setup.md).
 When done you will have a database with all tables created and with a table `CONFIGURATION` containing all configuration required for the cluster to start.
 
 1. Ensure that you meet the [requirements](hardware-and-software-requirements.md).
 1. [Create the database](database-configuration.md#database_creation).
 1. In case you use [Business data](define-and-deploy-the-bdm.md), [create a database for the Business Data](database-configuration.md#database_creation).
-1. Download a Bonita BPM [Tomcat bundle](tomcat-bundle.md) or a Bonita BPM [WildFly bundle](wildfly-bundle.md), and unzip it at some place of your choice.
-1. Edit file **`setup/database.properties`** and modify the properties to suit your databases (Bonita BPM internal database & Business Data database).
+1. Download a Bonita [Tomcat bundle](tomcat-bundle.md) or a Bonita [WildFly bundle](wildfly-bundle.md), and unzip it at some place of your choice.
```
Comment (educhastenier) [inline md/install-a-bonita-bpm-cluster.md]: restore `BPM` in 7.5

File: md/install-a-bonita-bpm-cluster.md [hunk 4]
```diff
@@ -169,25 +174,6 @@ Example:
...
-    UPDATE QRTZ_FIRED_TRIGGERS SET SCHED_NAME = 'BonitaClusteredScheduler';
-    UPDATE QRTZ_TRIGGERS SET SCHED_NAME = 'BonitaClusteredScheduler';
-    UPDATE QRTZ_SCHEDULER_STATE SET SCHED_NAME = 'BonitaClusteredScheduler';
-    UPDATE QRTZ_SIMPROP_TRIGGERS SET SCHED_NAME = 'BonitaClusteredScheduler';
-    UPDATE QRTZ_CALENDARS SET SCHED_NAME = 'BonitaClusteredScheduler';
-    UPDATE QRTZ_BLOB_TRIGGERS SET SCHED_NAME = 'BonitaClusteredScheduler';
-    UPDATE QRTZ_PAUSED_TRIGGER_GRPS SET SCHED_NAME = 'BonitaClusteredScheduler';
-    ```
-
-1. enable foreign keys on tables `qrtz_cron_triggers`, `qrtz_simple_triggers`, `qrtz_simprop_triggers` and `qrtz_triggers`
```
Comment (educhastenier) [inline md/install-a-bonita-bpm-cluster.md]: :+1:

Comment (JeremJR) [reply md/install-a-bonita-bpm-cluster.md]: so maybe that the target branch should be 7.4? and for other changes maybe even 7.3 (ie hazelcast port)

File: md/install-a-bonita-bpm-cluster.md (hunk 2 above)
Comment (ecuriotto) [reply md/install-a-bonita-bpm-cluster.md]: The port is configurable in hazelcast:
http://docs.hazelcast.org/docs/latest-development/manual/html/Setting_Up_Clusters/Other_Network_Configurations.html

I don't know how to change this port parameter in the Bonita files. If it's possible we can improve the doc. If it's not possible, provide the hardcode solution is better than ignore it in my opinion

Comment (educhastenier) [reply md/install-a-bonita-bpm-cluster.md]: ok, so please just add
"Hazelcast needs the port 5701 **(by default)**..." and it's ok for me

File: md/install-a-bonita-bpm-cluster.md (hunk 4 above)
Comment (tbouffard) [reply md/install-a-bonita-bpm-cluster.md]: agree with @JeremJR, changes may also be valid for Bonita 7.3

**Comment by educhastenier** [comment]
Ok, so please rebase on 7.4

**Comment by NathalieC** [comment]
@educhastenier : Do you agree this can be merged?

**Pull request merged**